### ISSUE TITLE
[Woo POS] TalkBack - Announce Changes to Payment States

### ISSUE THREAD

**Issue by malinajirka** [woocommerce/woocommerce-android#13547]
The app doesn't announce changes to the payment state with TalkBack enabled. This can be confusing for the users as their customer taps the card and seemingly nothing happens in the app - in reality, the app processes the payment and displays a success screen, but this is not announced by TalkBack.

**Comment by dangermattic** [comment]
🚫 Please add a type label (e.g. **type: enhancement**) and a feature label (e.g. **feature: stats**) to this issue.
